view_pdf_file/lib/constants.dart

```diff
@@ -0,0 +1,10 @@
+class Constants {
+  static final String pdfURL =
+      "https://raw.githubusercontent.com/nisrulz/flutter-examples/master/view_pdf_file/assets/Hello.pdf";
```

Comment:
Reference this directly in code. Once merged and on master, the URL will be functional and available via Github.